File: gateway/gateway-api/src/main/java/ru/gilko/gatewayapi/dto/statistic/ProfitableModelDto.java
package ru.gilko.gatewayapi.dto.statistic;

import lombok.AllArgsConstructor;
import lombok.Data;
import lombok.NoArgsConstructor;

@Data
@AllArgsConstructor
@NoArgsConstructor
public class ProfitableModelDto {
    private String model;
    private int proceeds;
    private int totalProceeds;
}

File: frontend/src/dtos/ProfitableModelDto.js
class ProfitableModelDto {
    model: string;
    proceeds: number;
    totalProceeds: number;

    constructor(model: string,
                proceeds: number,
                totalProceeds: number) {
        this.model = model;
        this.proceeds = proceeds;
        this.totalProceeds = totalProceeds;
    }
}

export default ProfitableModelDto;